docs/source/references/running.rst

```diff
@@ -0,0 +1,124 @@
+.. _running-the-app:
+
+Successfully Running the DRG
```

Comment:
How about just calling this "Run the DRG"? Obviously, you're giving the reader instructions to be successful. Ha.

Plus, "Run the DRG" alludes to these guys, of course: https://www.youtube.com/watch?v=vWaljXUiCaE